easybuild/easyconfigs/h/HPCX/HPCX-2.10.eb

```diff
@@ -0,0 +1,78 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'Tarball'
+
+name = 'HPCX'
+version = '2.10'
+
+homepage = 'http://www.mellanox.com/page/products_dyn?product_family=189&mtag=hpc-x'
+description = """The Mellanox HPC-X Toolkit
+is a comprehensive MPI and SHMEM/PGAS software suite
+for high performance computing environments"""
+
+toolchain = SYSTEM
+
+# By downloading, you accept the HPC SDK Software License Agreement (https://docs.nvidia.com/hpc-sdk/eula/index.html)
+# accept_eula = True
+source_urls = ['https://content.mellanox.com/hpc/hpc-x/v%(version)s/']
+sources = ['%(namelower)s-v%(version)s-gcc-MLNX_OFED_LINUX-5-redhat8-cuda11-gdrcopy2-nccl2.11-x86_64.tbz']
+checksums = ['973e301cdb02faf422d34063ea2abd9828dfd6c21ebe762c1d63fa8820dc028f']
+
+local_gccver = '11.2.0'
+dependencies = [
+    ('GCCcore', local_gccver),
```

Comment:
Why are you using the `SYSTEM` toolchain instead of `GCCcore/11.2.0`? With this dependency in place, it's compatibility with other modules will be as restricted as being in that GCCcore toolchain.